cli/cmd/vulnerabilty_exceptions.go

```diff
@@ -0,0 +1,354 @@
+
+    lacework vulnerability-exception create
+
+If you prefer to configure vulnerability exceptions via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Vulnerabilities > Exceptions.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(vulnerabilityExceptions)
+			}
+
+			var rows [][]string
+			for _, vuln := range vulnerabilityExceptions.Data {
+				rows = append(rows, []string{vuln.Guid, vuln.ExceptionName, vuln.ExceptionType, vuln.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "TYPE", "STATE"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework vulnerability exception by id
+	vulnerabilityExceptionShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Get vulnerability exception by id",
+		Long:  "Get a single vulnerability exception by it's vulnerability exception ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			var response api.VulnerabilityExceptionResponse
+			err := cli.LwApi.V2.VulnerabilityExceptions.Get(args[0], &response)
+			if err != nil {
+				return errors.Wrap(err, "unable to get vulnerability exception")
+			}
+			vuln := response.Data
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(vuln)
+			}
+
+			var groupCommon [][]string
+			groupCommon = append(groupCommon, []string{vuln.Guid, vuln.ExceptionName, vuln.ExceptionType, vuln.Status()})
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "TYPE", "ENABLED"}, groupCommon))
+			cli.OutputHuman("\n")
+			cli.OutputHuman(buildVulnerabilityExceptionsPropsTable(vuln))
+			return nil
+		},
+	}
+
+	// delete command is used to remove a lacework vulnerability exception by id
+	vulnerabilityExceptionDeleteCommand = &cobra.Command{
+		Use:   "delete",
+		Short: "Delete a vulnerability exception",
+		Long:  "Delete a single vulnerability exception by it's vulnerability exception ID.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			err := cli.LwApi.V2.VulnerabilityExceptions.Delete(args[0])
+			if err != nil {
+				return errors.Wrap(err, "unable to delete vulnerability exception")
+			}
+			return nil
+		},
+	}
+
+	// create command is used to create a new lacework vulnerability exception
+	vulnerabilityExceptionCreateCommand = &cobra.Command{
+		Use:   "create",
+		Short: "Create a new vulnerability exception",
+		Long:  "Creates a new single vulnerability exception.",
+		RunE: func(_ *cobra.Command, args []string) error {
+			if !cli.InteractiveMode() {
+				return errors.New("interactive mode is disabled")
+			}
+
+			err := promptCreateVulnerabilityException()
+			if err != nil {
+				return errors.Wrap(err, "unable to create vulnerability exception")
+			}
+
+			cli.OutputHuman("The vulnerability exception was created.\n")
```

Comment:
Should we print the GUID here? again, some commands print it and some other don't. (inconsistencies)

What is our standard?

```
cli/cmd/alert_rules.go:			cli.OutputHuman(fmt.Sprintf("The alert rule was created with GUID %s \n", response.Data.Guid))
cli/cmd/integration.go:			cli.OutputHuman("The integration was created.\n")
cli/cmd/lql_create.go:	cli.OutputHuman("The query %s was created.\n", create.Data.QueryID)
cli/cmd/policy_create.go:	cli.OutputHuman(fmt.Sprintf("The policy %s was created.\n", createResponse.Data.PolicyID))
cli/cmd/report_rules.go:			cli.OutputHuman("The report rule was created with GUID %s\n", response.Data.Guid)
cli/cmd/resource_groups.go:			cli.OutputHuman("The resource group was created.\n")
cli/cmd/vulnerabilty_exceptions.go:			cli.OutputHuman("The vulnerability exception was created.\n")
```

Reply:
I think it is useful to pass back the ID where we can